test/e2e/data/infrastructure-aws/kustomize_sources/nested-multitenancy-clusterclass/role.yaml

```diff
@@ -1,25 +0,0 @@
----
```

Comment:
How come this was deleted?It looks like its still being reference from the kustomization.yaml

Reply:
Does that make a difference? I think yaml is still getting applied.

Reply:
I think this was unintentional, but I am surprised how its working in E2E